parity-clib/examples/cpp/main.cpp

```diff
@@ -139,12 +120,11 @@ int parity_subscribe_to_websocket(void* parity) {
 		sessions.push_back(session);
 	}
 
-	while(cb.counter != 0);
-	std::this_thread::sleep_for(std::chrono::seconds(60));
+	std::this_thread::sleep_for(std::chrono::seconds(15));
 	for (auto session : sessions) {
 		parity_unsubscribe_ws(session);
```

Comment:
Using RAII would make this unnecessary, as `parity_unsubscribe_ws` would automatically be called.